### PR TITLE
Feat Display Swap Route Info

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

File: packages/web/utils/ssr.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const isBrowser = typeof window !== "undefined";
```
Comment (jonator) [inline packages/web/utils/ssr.ts:1]: Are these primitives really needed as separate files? Why not use them directly inline?

File: packages/web/hooks/use-measure.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  y: 0,
+  width: 0,
+  height: 0,
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+};
+
+function useMeasure<E extends Element = Element>(): UseMeasureResult<E> {
```
Comment (jonator) [inline packages/web/hooks/use-measure.ts:27]: Can we add a docs comment to explain this at a high level?

File: packages/web/utils/ssr.ts (hunk 1 above)
Comment (JoseRFelix) [reply packages/web/utils/ssr.ts:1]: I believe it's easier to read and understand thanks to the variable name. What do you think?

File: packages/web/hooks/use-measure.ts (hunk 2 above)
Comment (JoseRFelix) [reply packages/web/hooks/use-measure.ts:32]: Added 👍

**Pull request merged**